### PR TITLE
[doc] Add more details in Minikube walkthrough

### PR DESCRIPTION
**Pull request by kairen**
Add more details to to help developers clearly modify which file.

### CODE REVIEW

File: docs/minikube-walkthrough.md [hunk 1]
```diff
@@ -43,14 +43,23 @@ Again from the root of this repo run the following commands:
...
 cp ./sample-apiserver ./artifacts/simple-image/kube-sample-apiserver
 docker build -t <YOUR_DOCKERHUB_USER>/kube-sample-apiserver:latest ./artifacts/simple-image
-docker push <YOUR_DOCKERHUB_USER>/kube-sample-apiserver 
+docker push <YOUR_DOCKERHUB_USER>/kube-sample-apiserver
 ```
 
 ## Modify the the replication controller
 
-You need to change the ```imagePullPolicy``` to ```Always``` or ```IfNotPresent```.
+You need to modify the ```artifacts/example/rc.yaml``` file to change the ```imagePullPolicy``` to ```Always``` or ```IfNotPresent```.
```
Comment (sttts) [inline docs/minikube-walkthrough.md]: Can we link the file?

Comment (kairen) [reply docs/minikube-walkthrough.md]: @sttts Yes!! Update now.

**Comment by sttts** [comment]
Lgtm. Will hold that back until we have pushed the publisher results https://github.com/kubernetes/kubernetes/issues/53808.

**Comment by kairen** [comment]
@sttts No Problem! Thanks you for your reply.

**Pull request merged**